pkg/operator/ceph/cluster/mgr/mgr.go

```diff
@@ -294,16 +283,24 @@ func (c *Cluster) UpdateActiveMgrLabel(daemonNameToUpdate string, prevActiveMgr
 			if err != nil {
 				// In case of failure we report as 'active manager' the previous value, this way
 				// we force refreshing mgrRoleLabelName label next time this function is called
-				currActiveMgr = prevActiveMgr
```

Comment:
Is there a need to change this behavior? It will be simpler to test if we only get the cephcluster, instead of also changing the error handling.

Reply:
i've hand to change it because lookup of `prevActiveMgr` was moved to a separate method. But the code behaves exactly the same: we try to update all pods and later return error if at least one of them failed. 

Probably, it will be better to use Go [`errors.Join`](https://pkg.go.dev/errors#example-Join) to store pod update errors instead of boolean flag.

Reply:
With this change it seems `currActiveMgr` will never be updated [here](https://github.com/rook/rook/pull/14482/files#diff-8da7e11143cf4d0f27d87ee49d6f07282bd38b5008cd137ebc8445f11e237950R138-R142)

Reply:
it will. if there are no errors, current active mgr from ceph [will be returned](https://github.com/rook/rook/pull/14482/files#diff-8da7e11143cf4d0f27d87ee49d6f07282bd38b5008cd137ebc8445f11e237950R118).

Reply:
Ok, I see now how it returns `currActiveMgr` and updates the `activeMgr` var.